internal/support/scheduler/operators/intervalaction/get.go

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright 2019 VMware Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+
+package intervalaction
+
+import (
+	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
+)
+
+type CollectionExecutor interface {
+	Execute() ([]contract.IntervalAction, error)
+}
+
+type intervalActionLoadAll struct {
+	database IntervalActionLoader
+	limit    int
+}
+
+func (op intervalActionLoadAll) Execute() ([]contract.IntervalAction, error) {
```

Comment:
Adding to my other comment regarding honoring the `Configuration.Service.MaxResultCount` we should first check that the specified `limit` does not exceed the `Configuration.Service.MaxResultCount` before executing any database operations and simply return an `ErrLimitExceeded` error.

Reply:
That's true. However, I think this should be done in the http handler as doing the check directly here will introduce a cyclic dependency. It is possible to pass it as part of the struct and check inside. Up to you for final implementation.

Reply:
I am going to mark this resolved as I do not see a `limit` parameter in the current logic flow. Also, simply because a limit is specified at 1,000 there may only be 2 records returned, which would not exceed a MaxResultCount of 100.